src/node/server/serverPluginAssets.ts

```diff
@@ -1,9 +1,9 @@
 import { ServerPlugin } from '.'
-import { isImportRequest, isStaticAsset } from '../utils'
+import { isImportRequest } from '../utils'
 
 export const assetPathPlugin: ServerPlugin = ({ app }) => {
   app.use(async (ctx, next) => {
-    if (isStaticAsset(ctx.path) && isImportRequest(ctx)) {
+    if (ctx.resolver.isAssetRequest(ctx.path) && isImportRequest(ctx)) {
       ctx.type = 'js'
```

Comment:
The `resolver` should come from  the argument of  `assetPathPlugin`(`{app, resolver}`)

Reply:
👍 Fixed in 16dbf4d